### PR TITLE
httpify extracted and added tests

### PR DESCRIPTION
**Pull request by klorand**

### CODE REVIEW

**Comment by NiGGa** [comment]
Bár tudatosan bemergelhetnem. De finom sincs mi van benne.

**Comment by miklos-martin** [comment]
@NiGGa please use english in public

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 target/
 *.log
+.idea/
```
Comment (miklos-martin) [inline .gitignore:3]: this is not related to the project, as I've learnt it recently
I suggest you to check this out: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

File: src/main/scala/health/Checker.scala [hunk 2]
```diff
@@ -22,7 +24,10 @@ trait Checker extends HttpService {
       get {
         dynamic {
           onComplete(check) {
-            case Success(value) => complete("OK")
+            case Success(value) => {
+              value.foreach(println)
```
Comment (miklos-martin) [inline src/main/scala/health/Checker.scala:28]: Have you left this here by accident?

Comment (miklos-martin) [inline src/main/scala/health/Checker.scala:28]: Ah, OK, I see

**Comment by NiGGa** [comment]
@miklos-martin Please, suck my dick, thx!

File: src/test/scala/health/CheckerSpec.scala [hunk 3]
```diff
@@ -35,6 +41,23 @@ class CheckerSpec extends Specification with Specs2RouteTest with Checker {
...
+      }
+    }
+
+    "not httpify the endpoints if not already present" in {
+      set(Some("http://healthy,http://endpoints"), OK)
+
+      Get("/health") ~> sealRoute(checkRoute) ~> check {
+        observerdRequest.filter( _.uri.toString().startsWith("http://http")) should beEmpty
+      }
+    }
```
Comment (miklos-martin) [inline src/test/scala/health/CheckerSpec.scala:58]: would be nice to see a mixed case too

**Comment by miklos-martin** [comment]
@NiGGa you're too kind
